sd/zk/integration_test.go

```diff
@@ -18,13 +17,16 @@ var (
 
 func TestMain(m *testing.M) {
 	zkAddr := os.Getenv("ZK_ADDR")
-	if zkAddr == "" {
-		log.Fatal("ZK_ADDR is not set")
+	if zkAddr != "" {
+		host = []string{zkAddr}
 	}
-	host = []string{zkAddr}
+	m.Run()
```

Comment:
This subverts the intent, which is to explicitly not run the test unless ZK_ADDR is set.